content/docs/user-guide/basic-concepts/parameter.md

```diff
@@ -1,9 +1,9 @@
 ---
-name: 'Parameter Dependency'
-match: [parameter, parameters, param, params, hyperparameter, hyperparameters]
+name: 'Parameters'
+match: [parameter, parameters]
 tooltip: >-
-  Pipeline stages (defined in `dvc.yaml`) can depend on specific values inside
-  an arbitrary YAML, JSON, TOML, or Python file (`params.yaml` by default).
-  Stages are invalid (considered outdated) when any of their parameter values
-  change. See [`dvc params`](/doc/command-reference/params).
+  Simple values that your code can depend on, loaded from a a structured file
+  (`params.yaml` by default). DVC can track them as granular dependencies for
+  pipeline stages (defined in `dvc.yaml`). These are especially useful for
+  machine learning hyperparameter tuning. See `dvc params`.
```

Comment:
> > > simple - remove? (it can be array actually?)
> >
> > "simple values" is how we define parameters (in a few places already)
>
> not sure I understand :) I would remove it tbh

WDYT @dberenbaum ?

Cc @shcheklein

Reply:
Also, from https://github.com/iterative/dvc.org/pull/3414#pullrequestreview-1084598040

> > > We should be using more of a domain language, or have some connection to actual things besides DVC (can depend on hyperparameters, training parameters, any custom processing parameters ...)
> >
> > Hmmm. I can't use the word "parameters" to define "parameter".
>
> it's fine in this case to use "something parameters" to kinda clarify what we mean by parameters

Still not sure it's meaningful to use the word parameter in this tooltip definition.

Reply:
My take on it:

```
Hyperparameters or other configuration values used by your code
, loaded from a a structured file (`params.yaml` by default).
In pipeline stages (defined in `dvc.yaml`), DVC can track these granular 
dependencies as key-value pairs instead of tracking the entire file. DVC can
also compare these values between experiments, which is especially useful
for machine learning hyperparameter tuning. See `dvc params`.
```

Probably not great as is, but I tried to:
* Summarize that parameters are usually hyperparameters or some other code configuration.
* Contrast their dependency tracking to regular file dependencies.
* Mention how they are useful in DVC experiments.